src/elements/content-preview/ContentPreview.js

```diff
@@ -761,6 +761,11 @@ class ContentPreview extends React.PureComponent<Props, State> {
 
         if (selectedVersion) {
             setProp(fileOpts, [fileId, 'fileVersionId'], selectedVersion.id);
```

Comment:
It seems like the logic here is the same as what was in box-annotations, just moved up a level. Why isn't it working as-is?

Reply:
`bpx-content-preview` fetches file version and modifies `file.file_version` to be the file version that user is viewing. So they're always the same.

Reply:
Gotcha. Should we pass down the "current" file version id as a separate property rather than the derived boolean value?